Cargo.toml

```diff
@@ -24,6 +24,8 @@ serde = { version = "1.0.219", features = ["derive"] }
 sysinfo = {version = "0.33.1"}
 primitive-types = { version = "0.13.1" }
 toml ={ version = "0.8.22" }
+self_update = {version = "0.42.0"}
+tracing-appender = {version = "0.2"}
```

Comment:
The file logging should be in a separate branch as we dont wanna add it for this version, we wanna test it first